LEGO1/lego/legoomni/src/paths/legoextraactor.cpp

```diff
@@ -1,7 +1,24 @@
 #include "legoextraactor.h"
 
+#include "legosoundmanager.h"
+#include "misc.h"
+#include "mxmisc.h"
+#include "mxtimer.h"
+
 DECOMP_SIZE_ASSERT(LegoExtraActor, 0x1dc)
 
+// GLOBAL: LEGO1 0x100f31d0
+LegoWorld* g_unk0x100f31d0;
+
+// GLOBAL: LEGO1 0x100f31d4
+LegoAnimPresenter* m_assAnimP;
+
+// GLOBAL: LEGO1 0x100f31d8
+LegoAnimPresenter* m_disAnimP;
```

Comment:
These three variables just need to be initialized to NULL to pass the variable check